Description: Simplify build test
  The unit tests wre removed in Entr 5.3 and replaced with system tests that
  run git, tmux, vim etc. Such test are overkill to just validate that the
  built binary works. Instead of system_test.sh, simply run the entr binary
  once to validate it works on all architectures it was built on.
  .
  The more complex system_test.sh is a good candidate to be used in more
  extensive autopktests.
Author: Otto Kekäläinen <otto@debian.org>
Forwarded: not-needed
Updated: 2023-02-03

--- a/Makefile.bsd
+++ b/Makefile.bsd
@@ -6,7 +6,7 @@ CPPFLAGS += -DRELEASE=\"${RELEASE}\"
 all: versioncheck entr

 test: entr
-	./system_test.sh
+	ls entr.1 | ./entr -zns 'wc $$0'

 entr: entr.c ${EXTRA_SRC}
 	${CC} ${CFLAGS} ${CPPFLAGS} ${LDFLAGS} ${EXTRA_SRC} entr.c -o $@
